### ISSUE TITLE
[REQ] Option to change delay/velocity change when changing movement direction

### ISSUE THREAD

**Issue by kronflux** [fgsfdsfgs/perfect_dark#189]
Perfect Dark has always had sort of "spongy" movement, but it's far more noticeable on PC with WASD controls
for example, when you change direction from left(A) to right(D) while moving forward, there is a fairly noticeable delay in that change, which doesn't feel right in a PC FPS.
Obviously for remaining true to the original game and supporting controller input, this needs to remain in tact, but I'm hoping an option can be added to change the delay or disable it.

I would do a pull request, but a lot of the movement code is beyond my limited C knowledge.

**Comment by kronflux** [comment]
I imagine this would be as simple as changing bondwalk.c
Probably something like changing
`g_Vars.currentplayer->speedstrafe -= PALUPF(accelspeed * mult);`
and
`g_Vars.currentplayer->speedgo += accelspeed * g_Vars.lvupdate60freal * 2;`
to
`g_Vars.currentplayer->speedstrafe -= PALUPF(accelspeed * mult * g_PlayerStrafeAccelMultiplier);`
and
`g_Vars.currentplayer->speedgo += accelspeed * g_Vars.lvupdate60freal * g_PlayerStrafeAccelMultiplier;`
adding a config option in main.c for g_PlayerStrafeAccelMultiplier (or whatever)

**Comment by fgsfdsfgs** [comment]
I'd consider that a pretty significant gameplay change. Not to mention I myself don't really feel that it's very sluggish.
Maybe as an option in the future.

**Comment by kronflux** [comment]
yeah, fairly significant, although PC first person shooters definitely have accurate movement as a benefit over consoles, especially older consoles.
But yes, it's all a matter of personal preference, I often find myself getting stuck on doors because I didn't move quite as much as my brain thought I should have compared to pretty much every other first-person game I've played lol

Any chance you know what values would need to be affected? I could always try my hand at a personal commit, I'm just not sure where to start :P

**Comment by fgsfdsfgs** [comment]
Would probably be in `bwalkApplyMoveData` or one of the related functions in `bondmove.c` or `bondwalk.c`.

**Comment by Xii-Nyth** [comment]
...come to think of it im surprised theres no inertia exploits like in every other game with it 
by every I mean ace combat

can we get an april fools update where if you spin like a helicopter you fly?
